### PR TITLE
Dany/uniapp 29

### PR DESCRIPTION
**Pull request by azrield**

### CODE REVIEW

File: bin/test
Comment (yuvals2146) [inline bin/test:1]: what is it? this is empty file

File: package.json
Comment (yuvals2146) [inline package.json:1]: yarn should be installed global and not inside the package json AFAIK

File: src/test/db/loadPositionDataDB.test.js [hunk 1]
```diff
@@ -115,13 +115,13 @@ describe("loadPositionInit", () => {
   });
 
   test("should not load invalid position id form db", async () => {
-    expect(() => loadPosition(mockUnvalidPositionId)).rejects.toThrow(
+    expect(() => loadPosition(mockInvalidPositionId)).rejects.toThrow(
```
Comment (yuvals2146) [inline src/test/db/loadPositionDataDB.test.js:118]: melech 🤴

File: src/test/mocks.js [hunk 2]
```diff
@@ -85,6 +86,7 @@ const mockEtherPositionInfoDataTwo = {
   etherUsdExchangeRate: 1,
   ArbitUsdExchangeRate: 1,
   positionId: 2,
+  positionChain: 42161,
```
Comment (yuvals2146) [inline src/test/mocks.js:89]: adir

**Review by yuvals2146 — COMMENTED** [review]
Mushlam, just remove the empty file and the yarn and we are good to go

File: bin/test
Comment (azrield) [reply bin/test:1]: I've changed permissions on it:

```
diff --git a/bin/test b/bin/test
old mode 100755
new mode 100644
```

File: package.json
Comment (azrield) [reply package.json:1]: mmm.. so it's not part of the project? I've probably installed it both locally and globally. Thanks :)

File: bin/test
Comment (azrield) [reply bin/test:1]: mode 644 doesn't let one execute the script. you must have the 'x':

`👉 UniApp git:(UNIAPP-29) > ll bin              
total 24
drwxr-xr-x   5 danyazriel  staff  160 Sep  7 21:22 .
drwxr-xr-x  17 danyazriel  staff  544 Sep  7 21:44 ..
-rwxr-xr-x   1 danyazriel  staff  617 Sep  7 21:22 logger
-rwxr-xr-x   1 danyazriel  staff   91 Sep  8 05:13 start
-rwxr-xr-x   1 danyazriel  staff  611 Sep  7 21:22 test`

Comment (azrield) [reply bin/test:1]: removed. Thanks.

**Review by yuvals2146 — APPROVED** [review]

**Pull request merged**